src/main/java/com/iridium/iridiumskyblock/utils/PlayerUtils.java

```diff
@@ -60,7 +87,8 @@ public static void sendBorder(@NotNull Player player, @NotNull Island island) {
 
     public static void teleportSpawn(Player player) {
         World spawnWorld = Bukkit.getWorld(IridiumSkyblock.getInstance().getConfiguration().spawnWorldName);
-        if (spawnWorld == null) spawnWorld = Bukkit.getWorlds().get(0);
+        if (spawnWorld == null)
```

Comment:
Same here

Reply:
My code still forces it onto the next line but sure I _guess_